### ISSUE TITLE
Importing JSON files Offline Do Not Get the Default Values of The Properties

### ISSUE THREAD

**Issue by Alexbarp** [ni/niveristand-slsc-eds-custom-device#42]
In the existent version of SLSC EDS, we use JKI JSON parser, which work very well for a lot of applications but it can't obtain the "Default" values for the properties. So, all the channels created by the too will have "0" in it and you would need to go and change those values. You can avoid that by reading the JSON while the chassis is ON and then, export that configuration. This will make the parameters to persist (although this is another type of file).
However, SLSC NI R&D also created this tool: https://github.com/ni/slsc-capability-explorer that is able to read the JSON file offline and obtain the default values in the tree. This was not incorporated into the SLSC EDS because it required you to update the source-code to work in LV 2017. Since now we do not need to support previous versions, then maybe we incorporate such tool. @Karl-G1 @csjall

**Comment by rtzoeller** [comment]
See https://github.com/ni/niveristand-slsc-eds-custom-device/issues/38#issuecomment-582959663 about the potentially concerning licensing implications of this.

**Comment by Alexbarp** [comment]
@rtzoeller , @Karl-G1 , you are correct that this library is setup with GPL (copy left) within NI. However, I believe that this was setup with that by mistake since that is part of NI and its dependency is https://www.vipm.io/package/jdp_science_jsontext/ and this is BSD. It is very possible that this could be changed in the original code and we could notify this change. Please talk to 3rd party review commit if you want further help. The reason I am bringing this up is because this fix is one of the most annoying problems in working with offline SLSC capabilities and this could could help... Also, this code was create by our (marketing) request back then and it was unfortunate that they used GPL here... anyway...  :)

**Comment by rtzoeller** [comment]
@Alexbarp it looks like you authored the original commit in that repo which added the license file. Given the inactivity on the repo, you are likely the de facto owner - I recommend you work with the licensing team at NI to get the licensing issue sorted out as part of any request for this issue #42 to be fixed (which would need to be prioritized through @bdexni. At this time we don't have any priority associated with this improvement.
